streams/src/test/java/org/apache/kafka/streams/integration/EosIntegrationTest.java

```diff
@@ -174,13 +174,13 @@ public void createTopics() throws Exception {
     }
 
     @ParameterizedTest
-    @ValueSource(strings = {StreamsConfig.AT_LEAST_ONCE, StreamsConfig.EXACTLY_ONCE_V2})
+    @ValueSource(strings = {StreamsConfig.AT_LEAST_ONCE})
```

Comment:
Given that we have only a single parameter, we should remove it entirely, ie, both annotation, and the `eosConfig` parameter.